### DISCUSSION TITLE
Web Ui troubles

### DISCUSSION THREAD

**Discussion by BDHayden** [haugene/docker-transmission-openvpn#2129] [Q&A]
Hi, I am having troubles connecting to any of the web ui's for my containers I can get into them for a few seconds when starting the containers with docker-compose but I loose access shortly after

heres my config:

>     transmission-openvpn:
        image: haugene/transmission-openvpn
        container_name: transmission-openvpn
        restart: unless-stopped
        environment:
            OPENVPN_PROVIDER: PIA
            OPENVPN_CONFIG: ca_toronto,ca_montreal
            OPENVPN_USERNAME: ${PIA_USERNAME}
            OPENVPN_PASSWORD: ${PIA_PASSWORD}
            WEBPROXY_ENABLED: 'true'
            LOCAL_NETWORK: 10.0.0.0/24
            OPENVPN_OPTS: --inactive 3600 --ping 10 --ping-exit 60
            PUID: 1000  
            PGID: 1000 
            TRANSMISSION_DOWNLOAD_DIR: /downloads/Downloads
            TRANSMISSION_INCOMPLETE_DIR: /downloads/Downloads
            TRANSMISSION_INCOMPLETE_DIR_ENABLED: 'true'
            TRANSMISSION_WATCH_DIR: /downloads/torrent_links
            TRANSMISSION_WATCH_DIR_ENABLED: 'true'
            WEBPROXY_PORT: 34000
            HEALTH_CHECK_HOST: google.com
            net.ipv6.conf.all.disable_ipv6: 1
            net.ipv6.conf.default.disable_ipv6: 1
            net.ipv6.conf.lo.disable_ipv6: 1
            net.ipv4.ip_forward: 1
            OVERRIDE_DNS_1: 8.8.8.8
            OVERRIDE_DNS_2: 8.8.4.4
        dns:
            - 8.8.8.8
            - 8.8.4.4
        cap_add:
            - NET_ADMIN
        ports:
            - 9091:9091
            - 34000
        networks:
            - ${DOWNLOADS_NETWORK}
        volumes:
            - "${DOWN_DIR}/Downloads:/downloads"
            - 

> "${BASEPATH}/transmission/data:/data"

**Comment by pkishino** [comment]
LOCAL_NETWORK: 10.0.0.0/24
Are you sure this is correct??

**Reply by BDHayden** [reply]
Yes, this is a subnet i am not using. I have also tried changing that to another known subnet that is not in use.

**Reply by pkishino** [reply]
huh? Please read the wiki how to set this correctly..you need to set it your what you are actually using

**Reply by BDHayden** [reply]
This had been working for about a year untill i updated it and it stopped working.
